java/src/main/resources/template.properties

```diff
@@ -457,3 +457,7 @@ text.bigquery.output.dataset=
 text.bigquery.output.table=
 text.bigquery.output.mode=
 text.bigquery.temp.bucket=
+
+# Kafka TO BQ Dstream
+kafka.bq.consumer.group.id=group-id1
```

Comment:
Remove `group-id1`, it shouldn't be a default value if template property `kafka.bq.consumer.group.id` is not provided

Reply:
done